releasenotes/notes/fix_plot_histogram_number-a0a4a023dfad3c70.yaml

```diff
@@ -0,0 +1,9 @@
+---
+fixes:
+  - |
+    Fixed error in :func:`qiskit.visualization.plot_histogram` when using
+    number_to_keep smaller that the number of keys. The followint code will work now:
+    .. code-block:: python
+      from qiskit.visualization import plot_histogram
+      a = {'00': 3, '01': 5, '11': 8, '10': 11}
+      plot_histogram(a, number_to_keep=2)
```

Comment:
correct me if I'm wrong, but isn't the issue with multiple data-sets (i.e. ``plot_histogram(a, b, number_to_keep=2)``)? Your example doesn't show that at all.

Reply:
@drewrisinger This is not the issue with multiple datasets, but I saw in the code that multiple datasets must be supported, I checked that case and adapted the code for it. I'll add an example with multiple datasets as well.